chanzuckerberg/idseq-web
# [Maps] Add location_v2 loading to data discovery map display

### Description
- This completes the flow and add location_v2 loading to the map display.
- Matches the patterns for stats/dimensions and the left filtering sidebar still works as expected.
- Location endpoint returns the location info keyed by location_id for frontend lookups along with the list of sample_ids at that point. The main interaction (coming next) will be clicking a bubble -> seeing just those samples in the right sidebar.
- Most of the DiscoveryMap code was copied from MapPlayground.

### Demo
![May-28-2019 10-06-44](https://user-images.githubusercontent.com/5652739/58497252-65edb000-8130-11e9-812a-dda10b5a3591.gif)

### Test and Reproduce
- Load the branch with map keys, go to Data Discovery samples tab, click the map toggle, play around with the map and filters.
- See endpoint tests.
